ydb/core/tx/datashard/datashard_user_db.cpp

```diff
@@ -49,11 +49,11 @@ NTable::EReady TDataShardUserDb::SelectRow(
 }
 
 ui64 CalculateKeyBytes(const TArrayRef<const TRawTypeValue> key) {
-    return std::accumulate(key.begin(), key.end(), 0, [](ui64 bytes, const TRawTypeValue& value) { return bytes + value.IsEmpty() ? 1 : value.Size(); });
+    return std::accumulate(key.begin(), key.end(), 0ull, [](ui64 bytes, const TRawTypeValue& value) { return bytes + (value.IsEmpty() ? 1ull : value.Size()); });
```

Comment:
Please use a normal `for` loop here, these reducers are very hard to understand at a glance.

Reply:
Changed